book/src/rfc/rfc-0-generic-parameters.md

```diff
@@ -75,6 +75,117 @@ fn main() -> [Room; 4] {
 }
 ```
 
+## Builtin Examples
+Given the following function signatures for builtin functions:
+```rust
+fn from_bits(const N: Field, val: Field) -> [Field; N]
+fn from_bits(bits: [Bool; N]) -> Field
+```
+
+Calling the builtin functions in native code:
+```rust
+use std::to_bits;
+use std::from_bits;
```

Comment:
syntax should be:

```rust
use std::bits; // import
bits::to_bits(_); // usage
```

(this is to mimic golang)

Reply:
BTW I think we should have a syntax for builtins in general, so that `std::bits` can mix both builtins and native code. For example, the `bits` module could look like this:

```rust
fn not(bit: Bool) -> Bool {
  return 1 - bit;
}

builtin fn to_bits(const N: Field, val: Field) -> [Field; N];
```

with a `builtin` keyword?

Reply:
lgtm!